lib/containers/utils.pm

```diff
@@ -121,11 +129,8 @@ sub basic_container_tests {
     assert_script_run("$cmd_docker_container_prune");
     $output_containers = script_output("$runtime container ls -a");
     die("error: container was not removed: $cmd_docker_container_prune") if ($output_containers =~ m/test_2/);
+
     # Images can be deleted
-    my $images_to_delete = "alpine:$alpine_image_version hello-world opensuse/tumbleweed tw:saved";
-    $images_to_delete .= ' opensuse/leap' if (!check_var('ARCH', 's390x'));
-    #my $cmd_docker_rmi = "$runtime rmi $images_to_delete";
-    #my $output_deleted = script_output($cmd_docker_rmi);
```

Comment:
Why are you removing this?

Reply:
They are leftovers from the previous PR ^^ As I have done `rmi -a` there is no need of this variable!